settings.py

```diff
@@ -530,11 +530,6 @@ def JINJA_CONFIG():
 PUENTE = {
     'BASE_DIR': ROOT,
     'TEXT_DOMAIN': 'django',
```

Comment:
Mmm... I think `TEXT_DOMAIN` should have gone away in Puente 0.4, too. I'm not sure why I left it.